core/config/ocr2_config.go

```diff
@@ -9,7 +9,7 @@ import (
 
 type OCR2Config interface {
 	// OCR2 config, can override in jobs, all chains
-	GlobalOCR2ContractConfirmations() (uint16, bool)
+	OCR2ContractConfirmations() uint16
```

Comment:
This feels like a step backwards from the work that @connorwstein did

Reply:
I did this - I think the cleanest set path towards the ultimate goal of OCR2 chain specific (including non-evm) config support is the following:
1. All OCR2XXX env vars are global for all chains (thats this change). Chain specific config is done exclusively at the job spec level
2. We follow up with expanding chain specific defaults to non-evm chains

Reply:
yes - his is ocr1